indico/modules/events/sessions/util.py

```diff
@@ -156,23 +157,18 @@ def get_sessions_for_user(event, user):
 
 
 def serialize_session_for_ical(sess):
-    session_data = {
-        '_fossil': 'sessionMetadata',
+    from indico.modules.events.contributions.util import serialize_contribution_for_ical
+    from indico.modules.events.util import serialize_person_link
+    return {
+        '_fossil': 'sessionMetadataWithContributions',
         'id': sess.id,
         'startDate': sess.start_dt,
         'endDate': sess.end_dt,
-        'url': '',
+        'url': url_for('sessions.display_session', sess, _external=True),
```

Comment:
Why do you use `unicode` in L245 of `indico/modules/events/contributions/util.py` and here not? Is it really necessary btw?